libs/net/dal/Services/ReportService.cs

```diff
@@ -852,10 +857,9 @@ public IEnumerable<ReportInstance> GetLatestInstances(int id, int? ownerId = nul
                 var filterSettings = JsonSerializer.Deserialize<FilterSettingsModel>(section.Filter.Settings.ToJson(), _serializerOptions) ?? new();
 
                 // Modify the query to exclude content.
-                var excludeOnlyTheseContentIds = excludeContentIds.Any() && !sectionSettings.OverrideExcludeHistorical ? excludeContentIds : Array.Empty<long>();
-                var excludeAboveAndHistorical = sectionSettings.OverrideExcludeHistorical
-                    ? excludeOnlyTheseContentIds.AppendRange(excludeAboveSectionContentIds).Distinct().ToArray()
-                    : excludeOnlyTheseContentIds;
+                var excludeContentIds = excludeHistoricalContentIds.AppendRange(excludeReportContentIds).Distinct().ToArray();
```

Comment:
Fixed logic for excluding content from a linked report.

Fixed logic for excluding content from above sections.